src/instantiate.js

```diff
@@ -270,12 +270,13 @@ function translateAndInstantiate (loader, key, source, metadata, processAnonRegi
       metadata.load.format = detectLegacyFormat(source);
 
     var registered = false;
+    var address = (metadata.pluginLoad && metadata.pluginLoad.address) ? metadata.pluginLoad.address : key;
```

Comment:
I'm not sure if `metadata.pluginLoad.address` would sometimes be populated and we wouldn't want to use it.